cms/djangoapps/contentstore/video_storage_handlers.py

```diff
@@ -690,7 +690,10 @@ def _get_values(video, course):
         for attr in attrs:
             if attr == 'courses':
                 current_course = [c for c in video['courses'] if course_id in c]
-                (__, values['course_video_image_url']), = list(current_course[0].items())
+                if current_course:
+                    values['course_video_image_url'] = current_course[0][course_id]
```

Comment:
This seemed better in terms of clarity. I believe using list() and items() iterates twice as well which is worse for performance but that's not a big deal here.

Reply:
That makes sense. I am still slightly confused why you are indexing the dict with the course_id. Is that how you get the image url? If you could add a comment in to code to clarify this it would help me understand more.

Reply:
I thought the `video` looked like this from working it out backwards. And `current_course` is only populated with entries if `course_id in c`.
```
video = {
  'courses': [
    {
      'course_id_123': 'img.url',
      ...
    },
  ],
}
```

Reply:
Thanks for the example! It definitely clears up my confusion